### PR TITLE
Update Kafka REST proxy consumer API response in swagger spec

### PR DESCRIPTION
**Pull request by jihwang0709**
Update Kafka REST proxy consumer API response in swagger spec
- add timestamp and timestampType in ConsumeRecord

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json [hunk 1]
```diff
@@ -478,6 +478,17 @@
           "format": "int64",
           "description": "Offset for consumer record",
           "readOnly": true
+        },
+        "timestamp": {
```
Comment (yungezz) [inline specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json]: add properties in response is considered as breaking change, https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#new-property-added-to-response, suggest to add it in new API version.

Comment (jihwang0709) [reply specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json]: This feature is still in preview and no users yet. This is a recommendation from  a potential user who might consider to use the api if those properties exist. Hence, it is okay not to add new api version for this.

Comment (yungezz) [reply specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json]: thanks for update

**Comment by jihwang0709** [comment]
@yungezz Can you or someone review this PR?

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @jihwang0709  is the service change public on PROD? the PR is ready to merge?

**Comment by jihwang0709** [comment]
Yes.
Please merge this pr plz

**Pull request merged**